models/staging/stg_order.sql

```diff
@@ -1,8 +1,19 @@
 -- depends_on: {{ ref('stg_sales_data') }}
 
+{{
+    config(
+        schema='staging',
+        materialized='incremental'
```

Comment:
this should also include the `unique_key` and `incremental_strategy` configs in the case of status changes or line adds/subtracts (assuming order_date will not change)

```suggestion
        materialized='incremental',
        unique_key='order_number',
        incremental_strategy='merge',
        merge_update_columns=['order_line_number', 'status']  
```